src/com/dakl/Tests.java

```diff
@@ -16,6 +17,7 @@ void resetBoard() {
         b.displayBoard();
         b = CheckerBoard.blankBoard();
     }
+    // REVIEW : code duplication ? 
```

Comment:
This seems like a duplicate of a method with a similar name in Main. In general, it's OK to have "utility methods" in test code, but if it's the same code, you should just call the same method in Main.